### PR TITLE
rest: add request token endpoint

### PR DESCRIPTION
**Pull request by mvidalgarcia**
- Also fixes string duplications in tests

closes reanahub/reana-ui#93

### CODE REVIEW

File: reana_server/rest/users.py [hunk 1]
```diff
@@ -340,3 +351,95 @@ def _logout():
...
+        elif "access_token" in request.args:
+            user = get_user_from_token(request.args.get('access_token'))
+        user.request_access_token()
+        email_subject = f'[{REANA_URL}] Token request ({user.email})'
+        fields = ['id_', 'email', 'full_name', 'username', 'access_token',
+                  'access_token_status']
+        email_body = (
+            'New user access token request:\n\n' +
+            '\n'.join([f'{f}: {getattr(user, f, None)}' for f in fields]))
+        # TODO: change receiver
```
Comment (mvidalgarcia) [inline reana_server/rest/users.py:433]: How should we handle this?

Comment (diegodelemos) [reply reana_server/rest/users.py:433]: We could use [this variable](https://github.com/reanahub/reana/pull/309/files#diff-78c29e3abaa72a3c7f71bc5783c8c40bR38). We can rename it to `DEFAULT_EMAIL_RECEIVER` or `ADMIN_EMAIL`, what do you think?

**Comment by mvidalgarcia** [comment]
Reminder: I forgot to add audit logs

File: reana_server/rest/users.py (hunk 1 above)
Comment (mvidalgarcia) [reply reana_server/rest/users.py]: Done. We have to create the `ADMIN_EMAIL` env in the reana-server helm chart I guess.

**Pull request merged**